config.json

```diff
@@ -350,6 +350,16 @@
     },
     {
       "core": false,
+      "difficulty": 3,
+      "slug": "diamond",
+      "topics": [
+        "strings"
+      ],
+      "unlocked_by": null,
+      "uuid": "6045f586-06c8-a480-09df-3f71c8da339edb4f54b"
```

Comment:
I do not think this is valid. See https://github.com/exercism/swift/pull/304

Reply:
@masters3d thanks again for pointing this out. This UUID was most likely generated with an older version of configlet. The latest release [v3.6.1](https://github.com/exercism/configlet/releases) fixes this issue. For the record, configlet does not yet validate UUIDs when running `configlet lint`.